nix/marlowe-cardano/compose-spec.nix

```diff
@@ -2,7 +2,7 @@
 
 let
   inherit (pkgs) sqitchPg postgresql runCommand writeShellScriptBin writeText lib glibcLocales;
-  network = inputs.self.networks.__iogx__.preview; # TODO remove the __iogx__ when merged
+  network = inputs.self.networks.preview;
```

Comment:
Careful, make sure that this file is up-to-date with the latest `nix/compose.nix` that was deleted in this PR.